pkgs/applications/version-management/gittyup/default.nix

```diff
@@ -29,29 +29,35 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
-  cmakeFlags = [
-    "-DBUILD_SHARED_LIBS=OFF"
-    "-DUSE_SYSTEM_CMARK=ON"
-    "-DUSE_SYSTEM_GIT=ON"
-    "-DUSE_SYSTEM_HUNSPELL=ON"
-    # upstream uses its own fork of libgit2 as of 1.2.2, however this may change in the future
-    # "-DUSE_SYSTEM_LIBGIT2=ON"
-    "-DUSE_SYSTEM_LIBSSH2=ON"
-    "-DUSE_SYSTEM_LUA=ON"
-    "-DUSE_SYSTEM_OPENSSL=ON"
-    "-DENABLE_UPDATE_OVER_GUI=OFF"
-  ];
+  patches = [ ./0001-Fix-incorrect-order-of-argument-to-calloc-345.patch ];
+
+  cmakeFlags =
+    let
+      inherit (lib) cmakeBool;
+    in
+    [
+      (cmakeBool "BUILD_SHARED_LIBS" false)
+      (cmakeBool "USE_SYSTEM_CMARK" true)
+      (cmakeBool "USE_SYSTEM_GIT" true)
+      (cmakeBool "USE_SYSTEM_HUNSPELL" true)
+      # upstream uses its own fork of libgit2 as of 1.2.2, however this may change in the future
+      # (cmakeBool "USE_SYSTEM_LIBGIT2" true)
+      (cmakeBool "USE_SYSTEM_LIBSSH2" true)
+      (cmakeBool "USE_SYSTEM_LUA" true)
+      (cmakeBool "USE_SYSTEM_OPENSSL" true)
+      (cmakeBool "ENABLE_UPDATE_OVER_GUI" false)
+    ];
 
   nativeBuildInputs = [
     cmake
+    cmark
     ninja
     pkg-config
     wrapQtAppsHook
   ];
 
   buildInputs =
     [
-      cmark
```

Comment:
Actually I just checked again. It seems cmark is both required as build tool and regular library. So this needs to stay.